pageObject/homePage.js

```diff
@@ -3,12 +3,15 @@ let BasePage = require('../base/bagePage');
 let Input = require('../elements/input');
 let TextBox = require('../elements/textBox');
 let BaseElement = require('../base/baseElement');
+let DropDown = require('../elements/dropdown');
 
 let loginbuttonLocator = by.css('a.login');
 let searchTopInputLocator = by.css('#search_query_top');
 let submitSearchTopButtonLocator = by.css('.button-search');
 let searchResultsTextBoxLocator = by.css('.heading-counter');
 let firstSearchResultButton = by.css('.product_list.row > li:nth-child(1) h5 > a');
+let womenMenuDropdownLocator = by.xpath('//*[@id="block_top_menu"]/ul/li[1]');
+let womenMenuTShirtButtonLocator = by.xpath('//*[@id="block_top_menu"]//li[1]//li[1]//li[1]/a');
```

Comment:
optimize locator

Reply:
do not have an ideas how to fix it

Reply:
@germange check `.sfHover a[title="T-shirts"]`